docs/docs/how-tos/nebari-local.md

```diff
@@ -159,6 +159,15 @@ security:
         tag: sha-b4a2d1e
 ```
 
+### Increase fs watches
+
+Depending on your host system, you may need to increase the `fs.inotify.max_user_watches` and
+`fs.inotify.max_user_instances kernel parameters` if you see a `too many open files` in the logs of
```

Comment:
```suggestion
`fs.inotify.max_user_instances kernel parameters` if you see the error "too many open files" in the logs of
```
The syntax highlighting on the error message being the same as the config variables was confusing to read so I switched it up.